app/models/user.rb

```diff
@@ -3,7 +3,16 @@
 class User < ActiveRecord::Base
   has_secure_password 
   has_many :reviews
+  has_many :queue_items
 
   validates :email, presence: true, uniqueness: true
   validates :full_name, presence: true
+
+  def has_video_in_queue?(video)
+    !!video_queue_item(video)
```

Comment:
The only reason I used the additional `#video_queue_item(video)` method is because I use it elsewhere in the view.  The delete button needs a queue item to generate the right url, so I call `#video_queue_item(video)` on `current_user` to generate it.  So as long as have the method available to me, I may as well use it for this method, too.

Otherwise, yeah, that would be much better!